app/src/main/java/org/mozilla/focus/browser/BrowserFragment.kt

```diff
@@ -288,6 +287,13 @@ private class BrowserIWebViewCallback(
     override fun onBlockingStateChanged(isBlockingEnabled: Boolean) {}
 
     override fun onLongPress(hitTarget: IWebView.HitTarget) {}
+    override fun shouldInterceptRequest(url: String) {
+        launch(UI) {
```

Comment:
nit: make a comment that this may not be called from the UI thread and that's why we launch